### ISSUE TITLE
error message + visual bug when a flow node has two alternative sources

### ISSUE THREAD

**Issue by Lemente** [minetest/minetest#12550]
##### Minetest version
```
version 5.5.1
game: void
```

##### Summary
When providing a liquid_flow node with two sources in a list :
```
liquid_alternative_flowing = "void_liquid_two_source_one_flow:flow_blue_and_red",
liquid_alternative_source = {"void_liquid_two_source_one_flow:source_blue","void_liquid_two_source_one_flow:source_red"},
```
This generate an error message, and the flow doesn't connect properly to the sources.

Everything else behaves as expected.

![screenshot_20220717_203459](https://user-images.githubusercontent.com/27686967/179420422-1741ac20-6284-4990-904f-33f9e801b370.png)

##### Steps to reproduce

here is a custom mod to test the issue
[void_liquid_two_source_one_flow.zip](https://github.com/minetest/minetest/files/9128352/void_liquid_two_source_one_flow.zip)

give yourself some liquid nodes and place them:
/give singleplayer void_liquid_two_source_one_flow:source_red 10
/give singleplayer void_liquid_two_source_one_flow:source_blue 10

**Comment by sfan5** [comment]
Well what do you expect to happen here?
`liquid_alternative_source` is documented to take *exactly* one node name, of course this isn't going to work.

**Comment by Lemente** [comment]
I did this while making a tide mod.
I have multiple nodes all representing seawater, but with different ABMs/LBMs and purposes.
I do this to avoid having an entire ocean of ABMs firing every second.

It seemed natural to use the same flow node for all of them.


I tried not providing a list, and doing this instead:
source_a and flow_a linked together
then source_b also using flow_a

But this leads to source_b instantly turning into source_a, even when placed in a closed hole.
I was thinking of submitting this as a bug first. Should I? Or is there a reason for this behavior?
